content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -322,7 +322,7 @@ spec:
         kind: Ingress
         name: main-route
       target:
-        kind: Value
+        type: Value
```

Comment:
According to https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#metrictarget-v2beta2-autoscaling,
the target field is a MetricTarget object, which can have fields like averageUtilization, averageValue, type and value. 'kind' is obviously not the right property name.
This change makes sense to me.